src/Plugin/ContextReaction/JsonldTypeAlterReaction.php

```diff
@@ -89,9 +89,13 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
     $config = $this->getConfiguration();
     $form['source_field'] = [
       '#type' => 'select',
-      '#title' => $this->t('Source Field'),
+      '#title' => $this->t('Field containing RDF type information'),
       '#options' => $options,
-      '#description' => $this->t("Select the field containing the type predicates."),
+      '#description' => $this->t("In JSON-LD representations, allow the rdf:type to vary 
```

Comment:
As someone who hasn't mucked around with JSON-LD much this reads like a great improvement to me. Only thing I could suggest is potentially pointing out to some external documentation (if it exists) to further clarify.

Reply:
When i've put this in, i'd be happy to write such external docs, and provide a link.  "arcane backend process" is the only place i saw it described in the existing documentation repo.